src/views/v2/day/event/venue.php

```diff
@@ -9,14 +9,12 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
 
 // Setup an array of venue details for use later in the template
-$venue_details = tribe_get_venue_details( $event_id );
+$venue_details = tribe_get_venue_details( $event->ID );
```

Comment:
are we using tribe template tags to get venue details? or should we be getting it from the event object?

Reply:
Right now we don't have any other way of getting them besides building something similar to what the template tag does, but in the template.

@bordoni thoughts?